server/routes/printReport/printReport.ts

```diff
@@ -37,7 +41,21 @@ export default class PrintReportRoutes {
     const { user } = res.locals
     const { pdfMargins, adjudicationsUrl } = config.apis.gotenberg
     const adjudicationDetails = await this.reportedAdjudicationsService.getConfirmationDetails(adjudicationNumber, user)
-    const noticeOfBeingPlacedOnReportData = new NoticeOfBeingPlacedOnReportData(adjudicationNumber, adjudicationDetails)
+
+    const { reportedAdjudication, associatedPrisoner, prisoner } =
+      await this.decisionTreeService.reportedAdjudicationIncidentData(adjudicationNumber, user)
+    const offences = await this.decisionTreeService.getAdjudicationOffences(
```

Comment:
A bit confused by this - the offences are available from the API call. is this for the questions and answers?

Reply:
Yeah - all of this is to get the questions and answers (and fill them out with data - like prisoner names and whatnot)